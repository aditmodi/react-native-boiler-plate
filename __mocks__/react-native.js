import React, { Component } from 'react';

export const ReactNative = React;

ReactNative.StyleSheet = {
  create: function create(styles){
    return styles
  }
}

class View extends Component{
  render(){ return false; }
}

class ListView extends Component{
  static DataSource(){}
}

class AppRegistry{
  static registerComponent(){}
}

ReactNative.View = View;
ReactNative.ScrollView = ScrollView;
ReactNative.ListView = ListView;
ReactNative.Text = Text;
ReactNative.TouchableOpacity = View;
ReactNative.TouchableHighlight = View;
ReactNative.TouchableWithoutFeedback = View;
ReactNative.ToolbarAndroid = View;
ReactNative.Image = View;
ReactNative.AppRegistry = AppRegistry;
