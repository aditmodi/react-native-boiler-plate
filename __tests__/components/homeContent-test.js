import 'react-native';
import React from 'react';
import HomeContent from '../../app/components/homeContent';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
  const tree = renderer.create(
    <HomeContent/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
