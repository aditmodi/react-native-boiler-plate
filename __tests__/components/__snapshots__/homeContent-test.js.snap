// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "flex": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
  >
    Welcome to the HomePage
  </Text>
</View>
`;
